### PR TITLE
Fixed the Code according to Conventions (PEP-8).

### PR DESCRIPTION
**Pull request by yashLadha**
Fixes: #79
Signed-off-by: Yash Ladha <yashladhapankajladha123@gmail.com>

(cherry picked from commit ccd42a0)
Signed-off-by: Yash Ladha <yashladhapankajladha123@gmail.com>

### Checklist
- [x] My branch is up-to-date with upstream/develop branch.
- [x] Everything works and tested for Python 3.5.2 and above.

### Description
Changes the code to follow pep-8 convention.

### CODE REVIEW

**Comment by Mec-iS** [comment]
Cool, thanks. Once this is merged we can start improving the docstrings.

**Comment by yashLadha** [comment]
Sure @xadahiya :+1:

**Comment by Mec-iS** [comment]
@yashLadha You can confirm that tests pass smoothly?

**Comment by xadahiya** [comment]
@yashLadha submit the test logs to confirm everything is working after changes.

**Comment by yashLadha** [comment]
Actually when i try to build the library from `setup.py` the packages inside the root packages are not included in the package (build), because of which the tests are not running. @Mec-iS @xadahiya . Should i send a PR correcting the `setup.py` using setuptools.

**Comment by Mec-iS** [comment]
See #102 

Please check existing or closed issue when thinking about opening a new one.

**Comment by yashLadha** [comment]
I have seen that issue but there is no PR for the same. Shall I send a PR for it @Mec-iS

**Comment by Mec-iS** [comment]
Yeah. If there is no other PR dealing with the same issue

**Comment by yashLadha** [comment]
Sure @Mec-iS  :+1:

File: hydrus/hydraspec/doc_maker.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from typing import Any, Dict, Match, Optional, Tuple, Union
 
 
-def createDoc(doc: Dict[str, Any], HYDRUS_SERVER_URL: str=None, API_NAME: str=None) -> HydraDoc:
```
Comment (anubhav253) [inline hydrus/hydraspec/doc_maker.py]: Its Should have removed the earlier code here.

**Comment by anubhav253** [comment]
@yashLadha - I think you have missed few thingse merging develop in your branch here. the previous changes of @vddesai1998 is removed 
Details below please check once as i need to update my branch here
please check the comments here
https://github.com/HTTP-APIs/hydrus/pull/117/files#diff-02c90c20012d215257f523fc13bad720L48

**Comment by yashLadha** [comment]
I have merged only the conflicts and nothing else that contradict with my
PR.

**Comment by anubhav253** [comment]
@yashLadha - Got your point but if two different person changes the same line(both Might be implementing a different feature) then at the end it should contain changes of both person. which is not happening above please check once  Providing you a screenshot here
screenshot 98

**Comment by Mec-iS** [comment]
@yashLadha please merge the changes in this PR https://github.com/HTTP-APIs/hydrus/pull/96 you probably forgot to pull from `develop` or you didnt apply the changes to yoyr code

@anubhav253 thanks for pointing this out

**Comment by yashLadha** [comment]
Sorry for the convienience. Will send a Patch PR @Mec-iS

**Pull request merged**